pkg/model/iam/iam_builder_test.go

```diff
@@ -141,14 +141,14 @@ func TestS3PolicyGeneration(t *testing.T) {
 					}),
 					Resource: stringorslice.Slice([]string{
 						"arn:aws:s3:::bucket-name/cluster-name.k8s.local/addons/*",
+						"arn:aws:s3:::bucket-name/cluster-name.k8s.local/cluster.spec",
+						"arn:aws:s3:::bucket-name/cluster-name.k8s.local/config",
 						"arn:aws:s3:::bucket-name/cluster-name.k8s.local/instancegroup/*",
 						"arn:aws:s3:::bucket-name/cluster-name.k8s.local/pki/issued/*",
-						"arn:aws:s3:::bucket-name/cluster-name.k8s.local/pki/ssh/*",
 						"arn:aws:s3:::bucket-name/cluster-name.k8s.local/pki/private/kube-proxy/*",
 						"arn:aws:s3:::bucket-name/cluster-name.k8s.local/pki/private/kubelet/*",
-						"arn:aws:s3:::bucket-name/cluster-name.k8s.local/secrets/*",
-						"arn:aws:s3:::bucket-name/cluster-name.k8s.local/cluster.spec",
-						"arn:aws:s3:::bucket-name/cluster-name.k8s.local/config",
+						"arn:aws:s3:::bucket-name/cluster-name.k8s.local/pki/ssh/*",
+						"arn:aws:s3:::bucket-name/cluster-name.k8s.local/secrets/dockerconfig",
```

Comment:
Was the `dockerconfig` secret pushed up in a certain scenario? I couldn't find this secret for my clusters.

Reply:
I stumbled on [this](https://github.com/kubernetes/kops/blob/master/nodeup/pkg/model/secrets.go#L61-L74) and figured it was an option somewhere, though admittedly I didn't check if it was legacy or actually being used .. so i'll dig about

Reply:
It's actually fairly new I believe.  There's an explicit command to create it (kops create secret dockerconfig).  I don't know that I would actively recommend it, but there is nothing wrong with doing so.